recipes/shogun-cpp/meta.yaml

```diff
@@ -0,0 +1,79 @@
+{% set name = "shogun-cpp" %}
+{% set version = "6.0.0" %}
+{% set sha256 = "413da26db7fb57aa36c2981bf92d23817c0a29f22051b331dbe5b86e46119748" %}
+{% set blas_variant = "openblas" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: shogun_{{ version }}.tar.gz
+  url: https://github.com/shogun-toolbox/shogun/archive/shogun_{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+  patches:
+    - use-detected-lapack.patch  # From upstream: properly link to OpenBLAS
+
+build:
+  number: 0
+  features:
+    - blas_{{ blas_variant }}
+  skip:
+    - True  # [win]
+
+requirements:
+  build:
+    - toolchain
+    - arpack 3.5.*
+    - bzip2 1.0.*
+    - cmake
+    - ctags
+    - curl 7.52.*
+    - eigen
+    - glpk 4.61|4.62
+    - hdf5 1.8.18|1.8.18.*
+    - json-c 0.12|0.12.*
+    - libxml2 2.9.*
+    - lzo 2.09
+    - openblas 0.2.19|0.2.19.*
+    - protobuf 3.3.*
+    - snappy 1.1.4|1.1.6
```

Comment:
Ah, I see, I trusted the changelog without looking more closely. :)  Changed to just 1.1.4.